<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <title>Statistical Principles for Omics-based Clinical Trials</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="style.css">
</head>
<body>
<header>
<h3 class="date">October 2014</h3>

<h1 class="title">Statistical Principles for Omics-based Clinical Trials</h1>


<h2 class="author">Michael C Sachs</h2>
<p class="affilation"><em>National Cancer Institute</em></p>

</header>

<style type="text/css">
header#follow {

width: 160px;
position: fixed;
float: left;
margin-left: -160px;

}
</style>

<header id="follow">
<a href="//github.com/sachsmc/review-paper-cco/blob/gh-pages/Paper/review-paper-cco.md">Edit on github</a>
</header>


<p class="small"><strong>Abstract: </strong><em>High-throughput technologies enable the measurement of a large number of molecular characteristics from a small tissue sample. High-dimensional molecular information (referred to as omics data) offers the possibility of predicting the future outcome of a patient (prognosis) and predicting the likely response to a specific treatment (prediction). Embedded in the vast amount of data is the hope that there exists some signal that will enable practitioners to deliver therapy personalized to the molecular profile of a tumor, thereby improving health outcomes. The challenges are to determine that the omics assays are valid and reproducible in a clinical setting, to develop a valid and optimal omics-based test that algorithmically determines the optimal treatment regime, to evaluate that test in a powerful and unbiased manner, and finally to demonstrate clinical utility: that the test under study improves clinical outcome as compared to not using the test. We review the statistical considerations involved in each of these stages, specifically dealing with the challenges of high-dimensional, omics data.</em></p>



<h1 id="introduction">Introduction</h1>
<p>Omics technologies that generate a large amount of molecular data about a cancerous tumor have the potential to provide accurate predictions of a patient’s prognosis and predictions of their response to a specific treatment regime. The idea of omics-based biomarkers is that distinct tumor types can be identified using the multi-dimensional molecular data leading to treatment decisions personalized to that tumor type. An omics-based test can guide the decisions to treat or not to treat and help identify the particular therapy most likely to work. The challenge is to identify and demonstrate definitively that the use of an omics-based test improves clinical outcomes in a patient population.</p>
<p>An omics-based test can be used to predict a patient’s prognosis, which is their expected clinical outcome. A test that provides accurate predictions of prognosis, regardless of treatment, is referred to as a prognostic biomarker. A predictive omics test is one that accurately predicts disease outcomes with the application of specific interventions. Predictive markers are therefore useful for the selection among two or more treatment options. Statistically, a prognostic test is strongly associated with clinical outcome and a predictive test modifies the association between treatment and clinical outcome (interaction). High dimensional omics data can be used to identify specific molecular targets as potential mechanisms for drug development, however the use of omics technologies for drug development is beyond the scope of this review.</p>
<p>The path from development to definitively evaluating an omics-based test for prognosis or prediction of treatment response is long and arduous. Often, the end goal is to develop a test suitable for use in a clinical trial for guiding treatment. The oncology literature is full of reports that develop and/or evaluate omics-based tools for prognosis and prediction. Developing a simple test based on high-dimensional omics data can be complex and often uses novel statistical methods. Definitive evaluation of a prognostic or predictive test is costly and rife with methodological pitfalls. We aim to review such issues, giving you the resources to ask the right questions when critically weighing the evidence presented in a report of an omics-based study. Ultimately, as a practicing oncologist the question is: “Is this omics-based test something I want to use to improve patient care?”.</p>
<p>The long road to implementing a test in a practice starts with analytical validation, that is, demonstrating that the omics-based assay accurately and reproducibly measures the molecular quantities. After the assay performance is established comes the test development and preliminary evaluation. This involves reducing the high-dimensional data into a one-dimensional quantity that will be used to make a decision. This one-dimensional quantity is often a risk score: an estimate of the probability of a specific clinical outcome. It is necessary to establish the clinical validity of this risk score, that is, demonstrate that the risk score is independently associated with clinical outcome. Care must be taken to completely separate the development of the risk score from the evaluation, otherwise estimates can be optimistically biased. Finally, the risk score must be translated into a binary decision, often using a threshold. It remains to demonstrate that the use of the test to make this decision improves patient outcomes.</p>
<p>The following sections specify questions you should ask while reading a report of an omics-based clinical study. We review the importance of such questions, and common pitfalls to watch for. If you are reporting on an omics-based trial, answers to these questions should be made clear to the reader. Formal efforts to guide reporting have been developed, such as the REMARK checklist <span class="citation">(1)</span>, the GRIPS statement <span class="citation">(2)</span>, and a third guideline article that lacks an acronym <span class="citation">(3)</span>. Our review reflects these efforts through the readers’ lens.</p>
<h1 id="terminology">Terminology</h1>
<p>An omics-based test, or simple an <strong>omics test</strong>, is a mapping from the set of features on the omics assay to a single number. This number can be a binary value, such as good or poor prognosis, or it can provide a continuous scale, such as a risk score. It must be feasible to perform the test on an individual patient basis, by measuring the omics assay on the individual’s tissue. The assay generates lots of measurements, which we will refer to as <strong>features</strong>, and then fixed mathematical calculations are done to transform the many features into the single test value. Examples of such features are gene expression values, protein expression measurements, or genomic mutations.</p>
<p>Investigators determine the way that the mathematical calculations are done in the <strong>development phase</strong>. Often, there is a complete sample which is randomly allocated into <strong>development</strong> and <strong>validation</strong> samples. These are also sometimes referred to as <strong>training</strong> and <strong>test</strong> sets of samples. A report may cover only one of the two steps. At the end of the development phase, the model for the mathematical calculations is fixed and locked down.</p>
<p>That model is evaluated definitively in the <strong>validation</strong> phase in a completely independent sample. In order for the validation to be unbiased and definitive, it is imperative that no information from the validation sample leaks into the development phase. The validation should mimic realistic clinical use as much as possible, and that means that no further refinement to the test is allowed based on the observed results.</p>
<h1 id="what-is-the-intended-clinical-use">What is the intended clinical use?</h1>
<p>As with all clinical studies, the end goal is to improve patient care. Omics studies are no different, and a clear statement of the intended clinical use of the omics-test should be prominent. Carefully describing the context for the use of the assay determines the type of study needed to develop and validate it. The intended use of the assay also provides an overarching context in which to interpret the population under study, the assay measurements, and the statistical methods.</p>
<p>Omics-based tests in oncology generally are used for one of two clinical purposes: prognostic or predictive. A <strong>prognostic</strong> test is used to predict the likely clinical outcome of a patient. What is the clinical use of such a prediction? Often a prognosis is used to guide management of the disease. Patients with a very good prognosis may opt not to receive any treatment, while patients with a poor prognosis may opt for more aggressive treatment. An omics-based prognostic test that is currently used in practice is EndoPredict, which is used to predict recurrence in ER-positive, HER2-negative breast cancer <span class="citation">(4)</span>. For patients with a low risk of recurrence, it has been demonstrated that the risks of chemotherapy do not outweigh the benefits. Prognostic tests are clinically useful for guiding general disease management.</p>
<p><strong>Predictive</strong> tests are most useful for selecting patient populations for treatment with specific targeted therapies. This presumes the existence of a particular molecular targeted therapy. The predictive test is used to identify patients who will benefit from the targeted therapy. Predictive tests are generally based on only one or a few molecular characteristics that the therapy may target. For example, HER-2 is a gene that predicts a more aggressive form of breast cancer. Trastuzumab is a drug that specifically targets HER-2 and has been shown to be effective in HER-2 positive breast cancer <span class="citation">(5)</span>. While targeted therapies generally target only one molecular characteristic, omics assays can be used to identify molecular targets for less well-understood drugs. However, most successful targeted therapies have associated predictive tests that were developed based on the underlying biology rather than a broad search over a large number of molecular features <span class="citation">(6)</span>.</p>
<h1 id="what-is-the-patient-population-of-interest">What is the patient population of interest?</h1>
<p>Along with the intended clinical use, a report should have a clear statement of the intended population in which the test is being evaluated. This could be broad or quite specific. For the omics test to be useful, it must provide sufficient information above and beyond the standard of care in the target patient population. The distribution of the omics test and the expected benefit in the population should be clearly specified in advance.</p>
<p>The expected benefit of a new omics-based test could differ greatly by patient population. For instance, a prognostic test has more potential for benefit in stage 2 breast cancer than it does in stage 1 breast cancer, as the prognosis for stage 1 is already very good. Evaluating an omics-based test in a broad populations that encompasses multiple stages or multiple disease types can be difficult, as the test must provide more information beyond that provided by standard clinical and pathological factors.</p>
<h1 id="is-the-omics-assay-valid">Is the omics assay valid?</h1>
<p>Analytical validation of an assay involves evaluating the performance of the measurement in terms of accuracy, bias, and precision under a variety of conditions. Conditions are things like pre-analytic factors such as specimen quality, specimen collection, storage, and processing procedures, and technical aspects such as laboratory technician and batch effects from reagent lots or other assay materials. The high-dimensional nature of omics data makes it very difficult to assess each of the hundreds or thousands of outputs from a single assay. In developing a omics-based signature that only uses a subset of the components of a high-dimensional assay, one can analytically validate the final signature alone. However, prior to developing the signature, one must develop detailed standard operating procedures for specimen handling and processing to ensure a baseline level of validity.</p>
<p>Did the authors of the report state what type of specimens were used in the study? Can the test be applied to formalin-fixed paraffin embedded (FFPE) tissue, or only fresh-frozen? Most omics-based assays require a minimum percentage of tumor to be successful. A report should clearly state what criteria were used to screen tissue samples prior to running the assay. Generally this involves a criteria for the rejection of poor-quality specimens on the basis of percent tumor, percent necrosis, or some other marker of tissue quality.</p>
<p>Molecular assays can successfully be run on decades old FFPE tissue <span class="citation">(7)</span>. However, factors involved in the tissue processing and storage can impact the results <span class="citation">(8–10)</span>. Due to the high dimensionality of omics assays, a small amount of bias on each feature can translate into large errors when incorporating data from hundreds or thousands of features into a single continuous measurement. Therefore it is important to assess the impact of processing on the individual features in addition to the overall test.</p>
<p>In addition to processing and storage, technical aspects of an assay can impact the final results in a predictable way <span class="citation">(11,12)</span>. There could be technical effects, differences due to reagent lots, and other batch effects. Such batch effects are commonly recognized yet often ignored in high-dimensional assays <span class="citation">(13)</span>. Efforts should be made to measure the impact of these technical aspects and minimize them to the greatest extent possible. The way in which samples are assayed should be randomized to prevent confounding batch effects with the clinical outcome. Development and validation samples are sometimes run in the same batch or with the same lot of technical aspects. This does minimize batch effects, however, it can provide an overly optimistic assessment of the test, because in clinical use, running things in the same batch is not an option.</p>
<h1 id="on-what-samples-was-the-test-developed">On what samples was the test developed?</h1>
<p>Similar to developing criteria for rejection of tissue samples, in omics settings, criteria should be developed for the rejection of individual features (e.g. genes, proteins) prior to the development of the test. Features that do not pass the pre-specified quality metrics should be removed from consideration from the final test. Note that this feature processing step does not involve any clinical outcome measurements. As a concrete example, in the development of a gene expression based test, investigators may choose to exclude probe locations that have a dynamic range under some threshold, or probes for which only a small proportion of the samples had calls, or probes that have absolute expression levels below some threshold. Quality control steps like this can ensure a more robust a reproducible development of the test.</p>
<p>Don’t: confound technical factors with clinical outcomes. <span class="citation">(13,14)</span></p>
<p>Do: maintain strict separation between development and evaluation.</p>
<p>Do: cross validation if you have a data-sparse setting. <span class="citation">(15–18)</span></p>
<p>Don’t: use convoluted methods leading to overfitting.</p>
<h1 id="what-does-the-omics-test-do">What does the omics-test do?</h1>
<p>Once the analytical validity of the omics assay is established, the features are translated into a binary classification, a multi-category classification, or a continuous risk score. Carefully evaluate the methods used to perform this translation and ask how are the features of the omics assay translated into a clinically meaningful quantity?</p>
<p>Unfortunately, a common approach to developing prediction models is to use cluster analysis of omics features, ignoring the clinical outcome among the development samples. Cluster analysis is a class of methods that is used to partition samples into groups based on the similarities or differences among the omics features <span class="citation">(19)</span>. The meaning and number groups are not known in advance, but rather they are data dependent. Clustering is unsupervised in the sense that the groups discovery is done ignoring the true groups defined by the clinical outcome. The resulting clusters are not designed to provide valid information regarding a prognosis or prediction of response to therapy <span class="citation">(20)</span>. A common argument in favor of clustering is that it identifies biologically distinct groups. However, the groups are identified using a statistical algorithm and the biological relevance is only considered <em>post hoc</em>. For developing omics-based prognostic or predictive tests, it is better to use statistical methods which are designed to address those aims.</p>
<p>Often, there are more features measured than there are patients in the sample. In such high-dimensional settings, it is required to identify a subset of the features that will be used in the final multivariate mathematical model. There are two broad statistical approaches to this problem: <strong>filtering</strong> and <strong>regularization</strong>.</p>
<p>Filtering is a statistical approach where univariate methods are applied to each of the many omics features in turn. Typically, the univariate method involves estimating the association of the feature with the clinical outcome. Then, some criterion, which is chosen in advance or selected using cross-validation, is applied to the statistic to select a subset of features. For example, I am interested in developing a gene expression based test to predict clinical response to a new therapy. For each of the 1000 gene expression features that I have, I can compute a t-statistic comparing the expression levels for responders versus non-responders. I then filter out the genes with t-test p-values greater than 0.0001, and use the remaining ones in a multivariable logistic regression model to predict response. <span class="citation">(21)</span> describes a novel approach to filtering that is applied successfully to predict B-cell lymphoma subtypes using gene expression microarrays.</p>
<p>Regularization is an approach where all of the features in consideration are entered into a special multivariable statistical model for prediction of the clinical outcome, even if there are more features than samples. The special model includes a penalty component which encourages the model to throw out or downplay the impact of features that are not relevant. There are various types of penalty functions each with different properties, such as the LASSO <span class="citation">(22)</span>, the ridge penalty <span class="citation">(23)</span>, the elastic net <span class="citation">(24)</span>, and others <span class="citation">(19)</span>. Each type of penalty term contains at least one tuning parameter, which may be pre-specified or selected using cross-validation.</p>
<p>Each type of approach has its merits, and within each class there are a variety of specific models to choose from. In real applications, it is hard to determine what method will work best in advance. Instead of selecting a single model to use, multiple models can be averaged to improve prediction <span class="citation">(25)</span>. This approach, called Bayesian model averaging has proven successful in different applications, including prediction of cancer subtypes <span class="citation">(26)</span>. It is more common, however, to try out various different methods then select the one with the best performance. This is fine as long as the model selection is done entirely separated from the final validation sample. Leaking of information from the validation data into the model selection process can cause bias in insidious ways. Verify that the model selection and estimation process was done completely independently and locked down.</p>
<h1 id="on-what-samples-is-the-test-being-evaluated">On what samples is the test being evaluated?</h1>
<p>Do: define the clinical use <span class="citation">(27)</span></p>
<p>Study design: consider retrospective <span class="citation">(28)</span></p>
<p>Do: Design your study appropriately to answer the clinical question definitively <span class="citation">(29–40)</span></p>
<p>Do: Power your trial appropriately <span class="citation">(41,42)</span></p>
<p>Don’t: do partial resubstitution</p>
<h1 id="are-valid-methods-being-used-to-evaluate-the-test">Are valid methods being used to evaluate the test?</h1>
<p>Bad: IDI or net reclassification <span class="citation">(43,44)</span></p>
<p>Bad: Comparing AUCs for regression models <span class="citation">(45)</span></p>
<p>Good: comprehensive and pre-specified approach <span class="citation">(46)</span></p>
<h1 id="are-the-development-and-evaluation-samples-strictly-separated">Are the development and evaluation samples strictly separated?</h1>
<p>This issue has come up in previous sections, yet this error occurs so frequently that it needs to be highlighted in its own section. The evaluation sample for the assessment of a prognostic or predictive test needs to be completely independent from the development sample. This is especially true for omics-based tests, whose development is often complex and convoluted. Any information from the evaluation sample that leaks into the development sample can bias the results, making tests appear better than they truly are.</p>
<p>Leaking information between samples can happen in subtle ways. Sometimes, part of the model development process is done on the validation data again. This is called partial resubstitution <span class="citation">(20)</span>. For example, a common model development approach is to first filter a subset of 50 genes from a larger set of 450,000 based on their observed association with the outcome. Then, the 50 genes are put into a regression model to develop a single risk score. Occasionally, investigators will perform the filtering on the development sample and then re-estimate the regression model using the combined development and validation samples. This gives overly optimistic estimates of the performance of the algorithm. Partial resubstitution can be difficult to detect when the model development is more complex, and if cross-validation is used to estimate the performance.</p>
<p>In settings where relatively few samples are available, cross-validation is an efficient and valid approach to estimating performance <span class="citation">(47)</span>. The key point whether using the split sample approach or cross validation is that the entire model building process must be validated. Even informal checks of the model on the validation sample, such as viewing survival curve plots, prior to locking down the model can unknowingly cause bias. Therefore, once again we highlight the imperative that the validation sample be strictly separated from the <strong>entire</strong> model development process.</p>
<h1 id="concluding-remarks">Concluding remarks</h1>
<p>Do: follow reporting criteria <span class="citation">(1–3,48)</span></p>
<h1 id="references">References</h1>
<p></p>
<div class="references">
<p>1. Altman DG, McShane LM, Sauerbrei W, Taube SE. Reporting recommendations for tumor marker prognostic studies (rEMARK): Explanation and elaboration. BMC medicine. BioMed Central Ltd; 2012;10(1):51. </p>
<p>2. Janssens AC, Ioannidis J, Bedrosian S, Boffetta P, Dolan SM, Dowling N, et al. Strengthening the reporting of genetic risk prediction studies (gRIPS): Explanation and elaboration. European journal of clinical investigation. Wiley Online Library; 2011;41(9):1010–35. </p>
<p>3. McShane LM, Cavenagh MM, Lively TG, Eberhard DA, Bigbee WL, Williams PM, et al. Criteria for the use of omics-based predictors in clinical trials: Explanation and elaboration. BMC medicine. BioMed Central Ltd; 2013;11(1):220. </p>
<p>4. Filipits M, Rudas M, Jakesz R, Dubsky P, Fitzal F, Singer CF, et al. A new molecular predictor of distant recurrence in eR-positive, hER2-negative breast cancer adds independent information to conventional clinical risk factors. Clinical Cancer Research. American Association for Cancer Research; 2011;17(18):6012–20. </p>
<p>5. Fleeman N, Bagust A, Beale S, Dwan K, Dickson R, Director L, et al. Pertuzumab in combination with trastuzumab and docetaxel for the treatment of hER2 positive metastatic or locally recurrent unresectable breast cancer: A single technology appraisal. 2013; </p>
<p>6. Sawyers CL. The cancer biomarker problem. Nature. Nature Publishing Group; 2008;452(7187):548–52. </p>
<p>7. Iwamoto KS, Mizuno T, Ito T, Akiyama M, Takeichi N, Mabuchi K, et al. Feasibility of using decades-old archival tissues in molecular oncology/epidemiology. The American journal of pathology. American Society for Investigative Pathology; 1996;149(2):399. </p>
<p>8. Srinivasan M, Sedmak D, Jewell S. Effect of fixatives and tissue processing on the content and integrity of nucleic acids. The American journal of pathology. Elsevier; 2002;161(6):1961–71. </p>
<p>9. Maldegem F van, Wit M de, Morsink F, Musler A, Weegenaar J, Noesel CJ van. Effects of processing delay, formalin fixation, and immunohistochemistry on rNA recovery from formalin-fixed paraffin-embedded tissue sections. Diagnostic Molecular Pathology. LWW; 2008;17(1):51–8. </p>
<p>10. Specht K, Richter T, M<span>ü</span>ller U, Walch A, Werner M, H<span>ö</span>fler H. Quantitative gene expression analysis in microdissected archival formalin-fixed and paraffin-embedded tumor tissue. The American journal of pathology. Elsevier; 2001;158(2):419–29. </p>
<p>11. Pennello GA. Analytical and clinical evaluation of biomarkers assays: When are biomarkers ready for prime time? Clinical Trials. SAGE Publications; 2013;1740774513497541. </p>
<p>12. Isler JA, Vesterqvist OE, Burczynski ME. Analytical validation of genotyping assays in the biomarker laboratory. Future Medicine Ltd; 2007; </p>
<p>13. Leek JT, Scharpf RB, Bravo HC, Simcha D, Langmead B, Johnson WE, et al. Tackling the widespread and critical impact of batch effects in high-throughput data. Nature Reviews Genetics. Nature Publishing Group; 2010;11(10):733–9. </p>
<p>14. Soneson C, Gerster S, Delorenzi M. Batch effect confounding leads to strong bias in performance estimates obtained by cross-validation. PloS one. Public Library of Science; 2014;9(6):e100335. </p>
<p>15. McShane LM, Polley M-YC. Development of omics-based clinical tests for prognosis and therapy selection: The challenge of achieving statistical robustness and clinical utility. Clinical Trials. SAGE Publications; 2013;10(5):653–65. </p>
<p>16. Moons KG, Kengne AP, Woodward M, Royston P, Vergouwe Y, Altman DG, et al. Risk prediction models: I. development, internal validation, and assessing the incremental value of a new (bio) marker. Heart. BMJ Publishing Group Ltd; British Cardiovascular Society; 2012;98(9):683–90. </p>
<p>17. Moons KG, Kengne AP, Grobbee DE, Royston P, Vergouwe Y, Altman DG, et al. Risk prediction models: II. external validation, model updating, and impact assessment. Heart. BMJ Publishing Group Ltd; British Cardiovascular Society; 2012;heartjnl–l2011. </p>
<p>18. Polley M-YC, Freidlin B, Korn EL, Conley BA, Abrams JS, McShane LM. Statistical and practical considerations for clinical evaluation of predictive biomarkers. Journal of the National Cancer Institute. Oxford University Press; 2013;105(22):1677–83. </p>
<p>19. Hastie T, Tibshirani R, Friedman J, Hastie T, Friedman J, Tibshirani R. The elements of statistical learning. Springer; 2009. </p>
<p>20. Simon R, Radmacher MD, Dobbin K, McShane LM. Pitfalls in the use of dNA microarray data for diagnostic and prognostic classification. Journal of the National Cancer Institute. Oxford University Press; 2003;95(1):14–8. </p>
<p>21. Bair E, Tibshirani R. Semi-supervised methods to predict patient survival from gene expression data. PLoS biology. Public Library of Science; 2004;2(4):e108. </p>
<p>22. Tibshirani R. Regression shrinkage and selection via the lasso. Journal of the Royal Statistical Society Series B (Methodological). Blackwell Publishers; 1996;267–88. </p>
<p>23. Hoerl AE, Kennard RW. Ridge regression: Biased estimation for nonorthogonal problems. Technometrics. Taylor &amp; Francis Group; 1970;12(1):55–67. </p>
<p>24. Zou H, Hastie T. Regularization and variable selection via the elastic net. Journal of the Royal Statistical Society: Series B (Statistical Methodology). Blackwell Publishing Ltd; 2005;67(2):301–20. </p>
<p>25. Hoeting JA, Madigan D, Raftery AE, Volinsky CT. Bayesian model averaging: A tutorial. Statistical science. Institute of Mathematical Statistics; 1999;382–401. </p>
<p>26. Yeung KY, Bumgarner RE, Raftery AE. Bayesian model averaging: Development of an improved multi-class, gene selection and classification tool for microarray data. Bioinformatics. Oxford University Press; 2005;21(10):2394–402. </p>
<p>27. Mandrekar SJ, Sargent DJ. Predictive biomarker validation in practice: Lessons from real trials. Clinical trials. SAGE Publications; 2010;7(5):567–73. </p>
<p>28. Simon RM, Paik S, Hayes DF. Use of archived specimens in evaluation of prognostic and predictive biomarkers. Journal of the National Cancer Institute. Oxford University Press; 2009;101(21):1446–52. </p>
<p>29. Freidlin B, Korn EL. Biomarker enrichment strategies: Matching trial design to biomarker credentials. Nature Reviews Clinical Oncology. Nature Publishing Group; 2014;11(2):81–90. </p>
<p>30. Baker SG, Sargent DJ. Designing a randomized clinical trial to evaluate personalized medicine: A new approach based on risk prediction. Journal of the National Cancer Institute. Oxford University Press; 2010; </p>
<p>31. Baker SG, Kramer BS, Sargent DJ, Bonetti M. Biomarkers, subgroup evaluation, and clinical trial design. Discovery medicine. 2012;13(70):187–92. </p>
<p>32. Brannath W, Zuber E, Branson M, Bretz F, Gallo P, Posch M, et al. Confirmatory adaptive designs with bayesian decision tools for a targeted therapy in oncology. Statistics in medicine. Wiley Online Library; 2009;28(10):1445–63. </p>
<p>33. Denne JS, Pennello G, Zhao L, Chang S-C, Althouse S. Identifying a subpopulation for a tailored therapy: Bridging clinical efficacy from a laboratory-developed assay to a validated in vitro diagnostic test kit. Statistics in Biopharmaceutical Research. Taylor &amp; Francis; 2014;6(1):78–88. </p>
<p>34. Eng KH. Randomized reverse marker strategy design for prospective biomarker validation. Statistics in medicine. Wiley Online Library; 2014; </p>
<p>35. Freidlin B, Jiang W, Simon R. The cross-validated adaptive signature design. Clinical Cancer Research. AACR; 2010;16(2):691–8. </p>
<p>36. Freidlin B, McShane LM, Polley M-YC, Korn EL. Randomized phase iI trial designs with biomarkers. Journal of Clinical Oncology. American Society of Clinical Oncology; 2012;30(26):3304–9. </p>
<p>37. Freidlin B, Korn EL, Gray R. Marker sequential test (maST) design. Clinical Trials. SAGE Publications; 2013;1740774513503739. </p>
<p>38. Jiang W, Freidlin B, Simon R. Biomarker-adaptive threshold design: A procedure for evaluating treatment with possible biomarker-defined subset effect. Journal of the National Cancer Institute. Oxford University Press; 2007;99(13):1036–43. </p>
<p>39. Mandrekar SJ, Sargent DJ. Clinical trial designs for predictive biomarker validation: Theoretical considerations and practical challenges. Journal of Clinical Oncology. American Society of Clinical Oncology; 2009;27(24):4027–34. </p>
<p>40. Morita S, Yamamoto H, Sugitani Y. Biomarker-based bayesian randomized phase iI clinical trial design to identify a sensitive patient subpopulation. Statistics in medicine. John Wiley &amp; Sons, Ltd; 2014; </p>
<p>41. Mackey HM, Bengtsson T. Sample size and threshold estimation for clinical trials with predictive biomarkers. Contemporary clinical trials. Elsevier; 2013;36(2):664–72. </p>
<p>42. Peterson B, George SL. Sample size requirements and length of study for testing interaction in a 1<span class="math"> × </span>&lt; i&gt; k&lt;/i&gt; factorial design when time-to-failure is the outcome. Controlled clinical trials. Elsevier; 1993;14(6):511–22. </p>
<p>43. Hilden J, Gerds TA. A note on the evaluation of novel biomarkers: Do not rely on integrated discrimination improvement and net reclassification index. Statistics in medicine. John Wiley &amp; Sons, Ltd; 2013; </p>
<p>44. Pepe MS. Problems with risk reclassification methods for evaluating prediction models. American journal of epidemiology. Oxford University Press; 2011;kwr013. </p>
<p>45. Seshan VE, G<span>ö</span>nen M, Begg CB. Comparing rOC curves derived from regression models. Statistics in medicine. Wiley Online Library; 2013;32(9):1483–93. </p>
<p>46. Janes H, Brown MD, Huang Y, Pepe MS. An approach to evaluating and comparing biomarkers for patient treatment selection. The international journal of biostatistics. 2014;10(1):99–121. </p>
<p>47. Lee S. Mistakes in validating the accuracy of a prediction classifier in high-dimensional butsmall-sample microarray data. Statistical methods in medical research. SAGE Publications; 2008; </p>
<p>48. Bouwmeester W, Zuithoff NP, Mallett S, Geerlings MI, Vergouwe Y, Steyerberg EW, et al. Reporting and methods in clinical prediction research: A systematic review. PLoS medicine. Public Library of Science; 2012;9(5):e1001221. </p>
</div>
</body>
</html>